### PR TITLE
Autogenerate DITA Docs

### PR DESCRIPTION
**Pull request by jason-fox**
This PR adds a GitHub Action to generate the DITA Bootstrap Documentation on push.

Sample runs can be seen [here](https://github.com/jason-fox/dita-bootstrap/actions). The result of the current run can be seen [here](https://jason-fox.github.io/dita-bootstrap/index.html)

Screenshot 2021-06-03 at 08 38 33

### CODE REVIEW

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,33 @@
+name: CI
+'on':
+  push:
+#    branches:
```
Comment (jason-fox) [inline .github/workflows/ci.yml:4]: At the moment for testing all branches force a build. Uncomment as necessary to build on either develop and/or master. I think it probably should be develop.

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+            nodejs -v
+          plugins: |
+            https://github.com/jason-fox/fox.jason.extend.css/archive/master.zip
+            https://github.com/infotexture/dita-bootstrap/archive/develop.zip
+            https://github.com/jason-fox/fox.jason.prismjs/archive/master.zip
+          project: .github/dita-ot/html.xml
+      - name: Deploy HTML 🚀
+        uses: JamesIves/github-pages-deploy-action@3.7.1
+        with:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```
Comment (jason-fox) [inline .github/workflows/ci.yml:31]: This token may not be necessary depending upon the deploy type - see https://github.com/JamesIves/github-pages-deploy-action

**Review by infotexture — COMMENTED** [review]
Thanks, this looks pretty good. 

To avoid confusion, I don't think the generated output here should mimic the DITA-OT site color scheme in `theme.css`. 

We should probably use the same bright blue Bootstrap default color scheme in CI as generated by the plug-in's default output, so people know what they'll get out of the box.

— But there are a few things I've been meaning to adjust there too, so we should probably treat that as a separate issue rather than tweaking the theme in this PR.

File: .github/dita-ot/pdf.ditaval [hunk 3]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--  This file is part of the DITA Open Toolkit project. See the accompanying LICENSE file for applicable license.  -->
+<val>
```
[the remark this answers is not in the document]
Comment (jason-fox) [reply .github/dita-ot/pdf.ditaval]: File deleted - Fix cc6a788

File: .github/dita-ot/pdf.xml [hunk 4]
```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<?xml-model href="https://www.dita-ot.org/rng/project.rnc" type="application/relax-ng-compact-syntax"?>
+<project xmlns="https://www.dita-ot.org/project">
+  <deliverable id="pdf">
```
[the remark this answers is not in the document]
Comment (jason-fox) [reply .github/dita-ot/pdf.xml]: File deleted - Fix cc6a788

**Comment by jason-fox** [comment]
> We should probably use the same bright blue Bootstrap default color scheme in CI as generated by the plug-in's default output, so people know what they'll get out of the box.

 Fix cc6a788 .Custom CSS annotated and removed. Result can be seen [here](https://jason-fox.github.io/dita-bootstrap/

**Review by infotexture — APPROVED** [review]

File: .github/workflows/ci.yml (hunk 2 above)
Comment (infotexture) [reply .github/workflows/ci.yml:31]: Works as is. 🎉

**Comment by infotexture** [comment]
🚀 Results deployed to https://infotexture.github.io/dita-bootstrap/.

**Pull request merged**